serverless/knative_eventing/serverless-kn-trigger.adoc

```diff
@@ -0,0 +1,77 @@
+include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
+:context: serverless-kn-trigger
+
+[id="serverless-kn-trigger"]
+= Using triggers with Knative Eventing
+
+By default, all events which are sent to a channel or broker will be sent to all subscribers of that channel or broker.
+Using triggers allows you to filter events from a channel or broker, so that subscribers will only receive a subset of events based on your defined criteria.
+
+The Knative client provides a set of `kn trigger` commands that can be used to create and manage triggers.
+
+.Prerequisites
+Before you can use triggers, you will need:
+
+* Knative Eventing and `kn` installed.
+* An available broker (either the `default` broker, or one that you have created).
+* An available sink source.
+
+== Creating a trigger using `kn`
+
+.Procedure
+
+Create a trigger:
+----
+kn trigger create <TRIGGER-NAME> --broker <BROKER-NAME> --filter <KEY=VALUE> --sink <SINK>
+----
+
+== Deleting a trigger using `kn`
+
+.Procedure
+
+Delete a trigger:
+----
+kn trigger delete <TRIGGER-NAME>
+----
+
+== Describing a trigger using `kn`
+
+The `kn trigger describe` command prints information about a trigger.
+// add example of output
+
+.Procedure
+
+Print details of a trigger:
+----
+kn trigger describe <TRIGGER-NAME>
+----
+
+== Listing triggers using `kn`
+
+The `kn trigger list` command prints a list of available triggers.
+// add example of output
+
+.Procedure
+
+* Print a list of triggers:
++
+----
+kn trigger list
+----
+
+* You can also print a list of triggers in JSON format:
```

Comment:
@rhuss @navidshaikh should this be YAML as per the other PR for sources or is JSON correct in this case?

Reply:
JSON is correct in this case.

Reply:
Great, thanks!